content/docs/manifest.json

```diff
@@ -2,14 +2,15 @@
   "routes": [
     {
       "title": "cert-manager",
+
       "routes": [
         {
           "title": "Introduction",
           "path": "/docs/README.md"
         },
         {
-            "title": "Getting Started",
-            "path": "/docs/getting-started/README.md"
+          "title": "Getting Started",
+          "path": "/docs/getting-started/README.md"
```

Comment:
My editor altered  the  indentation using [prettier default  indentation of 2](https://prettier.io/docs/en/options.html#tab-width).